Package.swift

```diff
@@ -80,7 +80,7 @@ awsCCommonPlatformExcludes.append("source/arch/intel/asm")
 awsCCommonPlatformExcludes.append("source/arch/arm/asm")
 #endif
 
-var awsCIoPlatformExcludes = ["docs", "CODE_OF_CONDUCT.md", "codebuild"] + excludesFromAll
+var awsCIoPlatformExcludes = ["docs", "CODE_OF_CONDUCT.md", "codebuild", "PKCS11.md", "THIRD-PARTY-LICENSES.txt", "include/aws/io/private/pkcs11/v2.40/pkcs11.h", "include/aws/io/private/pkcs11/v2.40/pkcs11f.h", "include/aws/io/private/pkcs11/v2.40/pkcs11t.h"] + excludesFromAll
```

Comment:
don't need to exclude `include/private/pkcs11/...` anymore
those files are gone (moved under `source/...`)

Reply:
fixed